workflows/iuf/stages.yaml

```diff
@@ -24,106 +24,105 @@
 version: 0.1.0 # IUF version
 
 stages:
-    - name: process-media
-      type: product
-      operations:
-        - name: extract-release-distributions
-          local-path: operations/extract-release-distributions.yaml # this is relative to stages.yaml, which is contained in workflows/iuf/ in docs-csm
-          static-parameters: {} # any parameters that will be supplied statically to this operation.
+  - name: process-media
+    type: product
+    operations:
+      - name: extract-release-distributions
+        local-path: operations/extract-release-distributions.yaml # this is relative to stages.yaml, which is contained in workflows/iuf/ in docs-csm
```

Comment:
It feels like there should be a schema to this file. It would have multiple benefits:

* Prevent us from repeating documentation on each field throughout this file.
* Ensure it's always valid
* Make it clear to IUF developers how to edit this file and ensure it's still valid based on what the IUF operator expects.

Anyway, nothing for this PR to do since you're just changing whitespace.